.github/workflows/docker-release-build.yml

```diff
@@ -144,15 +144,15 @@ jobs:
         uses: actions/checkout@v2
       - name: 🐳 Docker meta
         id: docker_meta
-        uses: crazy-max/ghaction-docker-meta@v1
+        uses: docker/metadata-action@v3
         with:
           images: ${{ secrets.DOCKERHUB_REPO }}
-          label-custom: |
+          labels: |
             org.opencontainers.image.title=Gladys Assitant Production Image
-          tag-sha: true
-          tag-semver: |
-            {{raw}}
-            v{{major}}
+          tags: |
+            type=semver,pattern={{version}}
```

Comment:
We are losing the "v" here! 

See previous: 

<img width="599" alt="Screenshot 2021-09-16 at 10 43 17" src="https://user-images.githubusercontent.com/7365207/133540902-8845abb9-5915-4c91-8502-30544c1461ce.png">

See your build:

<img width="619" alt="Screenshot 2021-09-16 at 10 43 50" src="https://user-images.githubusercontent.com/7365207/133540925-41cc56ed-b9e9-422b-8bb7-4a9500b62e3e.png">

Do you confirm we are still pushing on latest as well ? thanks!

Reply:
Yes latest is automatic 
![image](https://user-images.githubusercontent.com/1528694/133565137-339720a4-22fd-4aa0-8f5e-9ebebe164516.png)